### DISCUSSION TITLE
Query

### DISCUSSION THREAD

**Discussion by ARAFATHYASIN** [Budibase/budibase#4099] [Help]
Hello Michael shanks,

I How you doing well,

I have a query ?

I am creating a single table which contains some data,(Records),the same way I am creating a N Number of tables which is called as a Database.
For example :
I am creating a single table for storing Single person Records.
Example:
For a single student, like (name, age, sex, address, mobile number, address details, previous school details,pervious marks sheet and every update of examination)in the basis of daily update.

The same way I am creating multiple tables for N number of students

How to merge all the single tables into a single project(Which contains N number of students details).Using our Budibase.

If I am calling the single project I need to retrieve all the students' records in a single table format.

A single table (Single project) contains N number of tables which contains multiple records of a students.

**Comment by mjashanks** [comment]
Hi Arafathysin.

So - you have multiple tables, and you want to merge them all into a single table?

You could
1. export all the data from the different tables (CSV)
2. update the CSV's column headers to match those of the target table
3. import them into the target table

Is this what you were asking for?

**Reply by ARAFATHYASIN** [reply]
Hello Mjashanks,

I have three table with different name 
[](url)
1)single Document
2)Revision
3)Project

1)single Document

It used to store the details of the student roll number, student name, father name,age,sex,address,marks statement and etc.

2)Revision

This table is used to update the news data's of the students like marks statement of every examination.

By using the roll number as a primary key, I am storing all other data's in this table.

3)Project 

This table is used to contain a data's of a Single class room students with the strength of 45 to 50 

i need to make a relationship table format using budibase DB.


While I am retrieving a only getting the information of first row in the Excel file.

I need in the format of I have attached in the below format.

[Book2.pdf](https://github.com/Budibase/budibase/files/7902519/Book2.pdf)

**Reply by mjashanks** [reply]
And which part of this are you having trouble with in Budibase ?

So, I assume that you have setup your tables with relationships follows
`Project has many Single Documents`
`Single Document has Many Revisions` 

Then in the design section, you could display data as follows

```
> Data Provider - Single Document
   > Repeater
      > Paragraphs and other elements to show single document details
      > Data Provider - Relationship > Single Document.Revisions
         > Table - listing revisions
```
